cluster/manifests/deployment-service/status-service-ingress.yaml

```diff
@@ -1,3 +1,4 @@
+# TODO: Remove after RouteGroup rollout
```

Comment:
You can already delete it, because we only do `apply` style. Maybe also add a deletions.yaml entry in [post apply](https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/cluster/manifests/deletions.yaml#L4-L5) to delete the ingress after routegroup apply

Reply:
Makes sense, changed.